apps/s_cb.c

```diff
@@ -26,10 +26,8 @@
 
 #define COOKIE_SECRET_LENGTH    16
 
-int verify_depth = 0;
-int verify_quiet = 0;
-int verify_error = X509_V_OK;
-int verify_return_error = 0;
+VERIFY_CB_ARGS verify_args = { 0 };
```

Comment:
If you're going to use an incomplete initializer, why not just use {} or even rely on the default initialization of global-scoped objects to zeros?